src/tenant/dto/update-tenant.dto.ts

```
@@ -0,0 +1,196 @@
+import { PartialType } from '@nestjs/mapped-types';
+import { CreateTenantDto } from './create-tenant.dto';
+
+export class UpdateTenantDto extends PartialType(CreateTenantDto) {}
+
+// src/tenant/tenant.service.ts
+import { Injectable, Logger, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository, DataSource, EntityManager } from 'typeorm';
+import { Tenant } from './entities/tenant.entity';
+import { CreateTenantDto } from './dto/create-tenant.dto';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'CreateTenantDto'. Consider to delete it or rename it.

'CreateTenantDto' is defined here:


(lint/suspicious/noRedeclare)